src/constants/default-settings.js

```diff
@@ -105,6 +105,11 @@ export const DIMENSIONS = {
   }
 };
 
+export const THEME = keyMirror({
```

Comment:
a think of description here will help people to understand they can use these values as kepler prop

Reply:
done